acmwebsite/lib/surveytypes.py

```diff
@@ -14,6 +15,10 @@ def from_post(self, form):
     def from_contents(self, contents):
         return literal_eval(contents)
 
+    def default(self):
```

Comment:
what is this `default` method and how should a programmer use it?

Reply:
Removed. It was necessary for sorting Python-side (comparing `None` with stuff blows up pretty quickly). If we do SQL sorting, then this is obsolete.